concepts/functions/.meta/config.json

```
@@ -3,6 +3,8 @@
     "glennj"
   ],
   "contributors": [
+    "kotp",
+    "IsaacG"
   ],
   "blurb": "Functions in bash programs."
```

```suggestion
  "blurb": "Functions in Bash programs."
```

Reason: This is the name of the language, so English says "capitalize first letter".  it also differentiates visually from the executable and the language itself.